docs/Evaluation.md

```diff
@@ -1,5 +1,43 @@
 # Evaluation 
 
+Independent analysis of whether a model can generalize from training data to new areas is critical for creating a robust workflow. 
+We stress that evaluation data must be different from training data, as neural networks have millions of parameters and can easily memorize thousands of samples. Therefore, while it would be rather easy to tune the model to get extremely high scores on the training data, it would fail when exposed to new images.
+
+To get an evaluation score, specify an annotations file in the same format as the training example above. The model will 
+```
+csv_file = get_data("OSBS_029.csv")
+root_dir = os.path.dirname(csv_file)
+results = model.evaluate(csv_file, root_dir, iou_threshold = 0.4)
+```
+
+The results object is a dictionary with keys, 'results',"recall","precision". Results is the intersection-over-union scores for each ground truth object in the csv_file.
```

Comment:
Can we rephrase this. I would think of this: " The returned object is a dictionary containing the three keys: `results`, `recall`, and `precision`."

`Not sure if the second part translates to this`

The result in the csv-file represents the `intersection-over-union` score for each ground truth object